Engine/plugin/agsplugin.cpp

```diff
@@ -575,6 +575,9 @@ void IAGSEngine::PlaySoundChannel (int32 channel, int32 soundType, int32 volume,
         newcha = my_load_midi(asset_name, (loop != 0)); break;
     case PSND_MOD:
         newcha = my_load_mod(asset_name, (loop != 0)); break;
+    case PSND_FLACSTREAM:
+    case PSND_FLACSTATIC:
+        newcha = my_load_ogg(asset_name, (loop != 0)); break;
```

Comment:
Should be my_load_flac?

Reply:
Yes, and yes. I'm sure I had renamed these correctly, but somehow...